packages/protocol/contracts/L1/libs/LibAuction.sol

```diff
@@ -130,27 +130,30 @@ library LibAuction {
         returns (bool provable, TaikoData.Auction memory auction)
     {
         if (blockId == 0) {
-            provable = false;
+            // provable = false;
```

Comment:
Can't this block remove the if block id == 0 check and instead make the else statement at the end "if block ID !== 0", since nothing is happening in the first statement anymore?